utils/password-hash.js

```diff
@@ -0,0 +1,6 @@
+const bcrypt = require('bcryptjs');
+
+exports.passwordHash = (password) => {
```

Comment:
we are not adding salt here check how admin are do login and fix this code else this new admin can't login

Reply:
salt is for database

Reply:
Okay I get you now